README.md

```diff
@@ -23,29 +23,23 @@ pip install -r requirements-dev.txt
 pre-commit install
 ```
 
- - Setup and run fastapi app:
- - In py3 virtual env do:
+Dev-env setup:
+--------------
 
+For running ubi-manifest related containers one can use provided docker-compose file.
+For succesfull running of celery tasks, it's required to properly update the config file ./conf/app.conf
+Also id any certs are requires they need to be present in ./conf/certs/
```

Comment:
In my opinion, you can just commit needed certs directly to the repo.  Certs aren't secrets after all, and if they're the ones I'm thinking of, there is precedent e.g. see https://github.com/dvdhrm/mgmt/blob/master/overlay/etc/pki/tls/certs/2015-RH-IT-Root-CA.pem .  Could cut down on the steps needed.

Reply:
ok, I wasn't sure about that, so I'll just put it there.